docs/semantics.md

```diff
@@ -95,7 +95,9 @@ The negative result for ```foo/``` will be cached, but that only helps with the
 To alleviate this, Cloud Storage FUSE supports a "type cache" on directory inodes. When type cache is enabled, each directory inode will maintain a mapping from the name of its children to whether those children are known to be files or directories or both. When a child is looked up, if the parent's cache says that the child is a file but not a directory, only one Cloud Storage object will need to be stated. Similarly if the child is a directory but not a file.
 
 The behavior of type cache is controlled by the following flags/config parameters:
-1.  Type-cache TTL: It controls the duration for which Cloud Storage FUSE allows the kernel to inode type attributes. It can be set in one of the following two ways.
+1. Type-cache capacity: This is configurable at per-directory level by setting `metadata-cache: type-cache-max-size-mb-per-dir` in config-file. This is the maximum size of type-cache per-directory in MiBs. By default, this is set at 32, which roughly equates to about a million entries (based on 32 bytes per [type-cache entry](https://github.com/GoogleCloudPlatform/gcsfuse/blob/abbdd1013f251ef078f405a632bc3b64a4ed48ab/internal/fs/inode/type_cache.go#L56)).
```

Comment:
`Type-cache capacity` needs to be removed now as it won't be published to users. Please confirm.

Reply:
old outdated commemnt.